boards/phytec/phyboard_atlas/board.cmake

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2025 PHYTEC America, LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CONFIG_SOC_MIMXRT1176_CM7 OR CONFIG_SECOND_CORE_MCUX)
+ board_runner_args(pyocd "--target=mimxrt1170_cm7")
+ board_runner_args(jlink "--device=MIMXRT1176xxxA_M7")
+
+ board_runner_args(linkserver "--device=MIMXRT1176xxxxx:MIMXRT1170-EVK")
+ board_runner_args(linkserver "--core=cm7")
+elseif(CONFIG_SOC_MIMXRT1176_CM4)
+ board_runner_args(pyocd "--target=mimxrt1170_cm4")
+ # Note: Please use JLINK above V7.50 (Only support run cm4 image when debugging due to default boot core on board is cm7 core)
+ board_runner_args(jlink "--device=MIMXRT1176xxxA_M4")
+
+ board_runner_args(linkserver "--device=MIMXRT1176xxxxx:MIMXRT1170-EVK")
+ board_runner_args(linkserver "--core=cm4")
+endif()
+
+if(NOT SYSBUILD)
+  board_runner_args(jlink "--no-reset")
+  board_runner_args(linkserver "--no-reset")
+endif()
```

Comment:
this should not be doing sysbuild exceptions, what is this meant to do?

Reply:
I honestly just synced that from NXP's board.cmake file: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/mimxrt1170_evk/board.cmake#L30

I would like to keep our board similar to NXP's.

Reply:
@EmilioCBen can you explain this please, there should be no knowledge or care if sysbuild is being used in a board.cmake file, and this needs to go from your board

Reply:
@nordicjm I just reverted this change from this board.cmake file.

Reply:
@nordicjm The if statement originally comes from this PR files[87423](https://github.com/zephyrproject-rtos/zephyr/pull/87423/files)

The use case was that a user wanted to debug something that was flashed to ram but did not want to remove the image that was inside the flash. I added the sysbuild check when anticipating that sysbuild usually requires a flash. I am unable to test this myself on the 1170 as sysbuild does not seem to work or has not prior been enabled.

I have a removal PR for this check right here. [96038](https://github.com/zephyrproject-rtos/zephyr/pull/96038)
..Sorry about the delay response, I thought I might have been able to enable the sysbuild as well..